.github/workflows/build_and_test.yml

```diff
@@ -606,9 +608,22 @@ jobs:
         run: |
           docker pull 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node-${{ matrix.version }}:${{needs.tag.outputs.build-tag}}
 
+      - name: Downloading VM informant version ${{ env.VM_INFORMANT_VERSION }}
+        run: |
+          curl -L https://github.com/neondatabase/autoscaling/releases/download/${{ env.VM_INFORMANT_VERSION }}/vm-informant -o vm-informant
+          chmod +x vm-informant
+
+      - name: Adding VM informant to compute-node image
+        run: |
+          ID=$(docker create 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node-${{ matrix.version }}:${{needs.tag.outputs.build-tag}})
```

Comment:
what do you think about adding this into `Dockerfile.compute-node-v1[45]`?

Reply:
finally, we will have only VMs, and this binary is lightweight.

Reply:
We are opensource and I think people mostly run neon locally in docker or in their k8s clusters. NeonVM without kvm (bare-metal hosts) too slow.

Reply:
I also think the `vm-informant` shouldn't be included in the non-VM compute node images, for two reasons:
1. In general, we shouldn't run programs in environments we don't use them
2. The `vm-informant` binary is ~40MB. That's not _huge_, but it's not insignificant either.

Reply:
sounds good